cli/src/commands/deploy_configs.py

```diff
@@ -6,12 +6,17 @@ def get_name(self):
         return "deploy"
 
     def get_description(self):
-        return "Deploy configs"
+        return "Deploy a config"
```

Comment:
Hmmm so there is deploy_config.py and deploy_configs.py - but deploy_configs.py is deploying a single config.

How does both differ?

Reply:
I have renamed it. (deploy_config.py didn't exist before)